src/main/java/com/my/FoodTruckBackend/entree/EntreeRepository.java

```diff
@@ -0,0 +1,41 @@
+package com.my.FoodTruckBackend.entree;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.dao.DuplicateKeyException;
+import org.springframework.http.HttpStatus;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+import org.springframework.web.server.ResponseStatusException;
+
+@Repository
+@RequiredArgsConstructor
+@Slf4j
+public class EntreeRepository {
+    private final JdbcTemplate jdbcTemplate;
+
+    public Entree createNewEntree(EntreeRequestBody entreeRequestBody) {
+        String sql = "INSERT INTO entree(name,description,price) VALUES(?,?,?) RETURNING *";
+        try {
+            return jdbcTemplate.queryForObject(
+                sql,
+                new BeanPropertyRowMapper<>(Entree.class),
+                entreeRequestBody.getName().trim(),
+                entreeRequestBody.getDescription().trim(),
+                entreeRequestBody.getPrice()
+            );
+        } catch (DuplicateKeyException duplicateKeyException) {
+            String errorMessage = "Entree:" + entreeRequestBody.getName() + "could not be added";
```

Comment:
Could we tell the reason why the new entree could not be added? For example : 
"This Entree already exists"